### PR TITLE
docs: minor fixes to how to work on coding challenges

### PR DESCRIPTION
**Pull request by jcjc2019**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Some minor doc updates to clarify the meanings for the docs on coding challenges, including some minor changes to clarify the video solution.

### CODE REVIEW

**Review by Sembauke — APPROVED** [review]

File: docs/how-to-work-on-coding-challenges.md [hunk 1]
```diff
@@ -251,7 +252,7 @@ We track how long it takes for campers to solve challenges and use this informat
 
 Each challenge should teach exactly one concept, and that concept should be apparent from the challenge's name.
 
-We can reinforce previously covered concepts through repetition and variations - for example, introducing h1 elements in one challenge, then h3 elements a few challenges later.
+We can reinforce previously covered concepts through repetition and variations, for example, introducing h1 elements in one challenge, then h3 elements a few challenges later.
```
Comment (jeremylt) [inline docs/how-to-work-on-coding-challenges.md:255]: I would split this sentence up. I don't think that putting a comma here is correct grammar. As a general rule, commas should be used sparingly. They aren't replacements for conjunctions.

> We can reinforce previously covered concepts through repetition and variation. For example, we can introduce h1 elements in one challenge and h3 elements a few challenges later.

File: docs/how-to-work-on-coding-challenges.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 Our goal is to develop a fun and clear interactive learning experience.
 
-Designing interactive coding challenges is difficult. It would be much easier to write a lengthy explanation or to create a video tutorial. But for our core curriculum, we're sticking with what works best for most people - a fully interactive, video game-like experience.
+Designing interactive coding challenges is difficult. It would be much easier to write a lengthy explanation or to create a video tutorial. But for our core curriculum, we're sticking with what works best for most people -- a fully interactive, video game-like experience.
```
Comment (jeremylt) [inline docs/how-to-work-on-coding-challenges.md:5]: Do we have a style guide for this sort of thing? Using -- in a block of text is a bit unusual in my experience

**Comment by raisedadead** [comment]
We are officially on Node.js 18 now. We have changed the merge requirements accordingly.

You will need to rebase this PR against the main branch OR Click the "Update branch" button to get the new checks going.

Thanks for your contribution.

**Comment by raisedadead** [comment]
Hi, 

We are closing old issues/pull requests that we think are stale. If you still need help or are fixing this, please open a new issue/pull request.

Thanks for your understanding. 

Happy contributing.

**Pull request closed without merging**